### ISSUE TITLE
UPDATE FOR BETA / PTR 11.0 WAR WITHIN

### ISSUE THREAD

**Issue by lloskka** [jaredbwasserman/drift#64]
Good morning.

Is there any ETA on an update for war within ptr or beta ?!

Cheers.

**Comment by webdevdave** [comment]
Came here for the same <3

**Comment by SelfWinding** [comment]
I would love to see an update <3

**Comment by Mlarro** [comment]
I have been using drift for, i dont know how long. Only now when it isnt working i realize how much i use it. Would love to see an update for the pre-patch/beta/ptr

**Comment by devwrk** [comment]
While you wait, you can use MoveAny. This is the addon I was using before switching to Drift.

**Comment by lnevo** [comment]
> While you wait, you can use MoveAny. This is the addon I was using before switching to Drift.

MoveAny is good and will work, but it is also a resource hog and the reason I switched to Drift. Another bump for support for 11.x!

**Comment by AlanDugger** [comment]
Here's a work around until it's official fixed.

Navigate to your Warcraft of Warcraft Folder, then into _retail_ then into Interface then Addons, then Drift.
You should see a file called DriftOptions.lua which you can open with a Notepad, Notepad++, or other text editor
Go down to specifically line 95 which should look like this:

InterfaceOptions_AddCategory(DriftOptionsPanel.optionspanel)

Change it by adding 2 dashes in front so that it now looks like this:
--InterfaceOptions_AddCategory(DriftOptionsPanel.optionspanel)

Save and load your game.

**Comment by devwrk** [comment]
> Change it by adding 2 dashes in front so that it now looks like this: `--InterfaceOptions_AddCategory(DriftOptionsPanel.optionspanel)`
> 
> Save and load your game. This should load your previous settings, but won't let you do new movements.

Tested and mostly works for me too. Found the Professions window could not move anymore.

**Comment by halloaloe** [comment]
Another hand raised for a WW update, please!

**Comment by tusharsaxena** [comment]
Working edit (including saving settings across sessions) - change DriftOptions.lua Line 95 from

`
	InterfaceOptions_AddCategory(DriftOptionsPanel.optionspanel)
`
to

```
	--InterfaceOptions_AddCategory(DriftOptionsPanel.optionspanel)
	if InterfaceOptions_AddCategory then
		InterfaceOptions_AddCategory(DriftOptionsPanel.optionspanel)
	else
		local category, layout = Settings.RegisterCanvasLayoutCategory(DriftOptionsPanel.optionspanel, DriftOptionsPanel.optionspanel.name);
		Settings.RegisterAddOnCategory(category);
	end
	self.optionsFrame = optionsFrame

```

**Comment by add1kted2ka0s** [comment]
Working edit (including saving settings across sessions). Change DriftOptions.lua Line 95 from

`InterfaceOptions_AddCategory(DriftOptionsPanel.optionspanel)`

to

```
	--InterfaceOptions_AddCategory(DriftOptionsPanel.optionspanel)
	if InterfaceOptions_AddCategory then
		InterfaceOptions_AddCategory(DriftOptionsPanel.optionspanel)
	else
		local category, layout = Settings.RegisterCanvasLayoutCategory(DriftOptionsPanel.optionspanel, DriftOptionsPanel.optionspanel.name);
		Settings.RegisterAddOnCategory(category);
	end
	self.optionsFrame = optionsFrame

```

**Comment by devwrk** [comment]
The professions frame was still not moveable (the one with all the professions)

Change Drift.lua
Add the following after line 519 (inside the list at the end)

```
	["ProfessionsBookFrame"] = {
		DriftDisabledBy = "windowsDisabled",
	},
``` 

Should look like this
```
	},
	["ProfessionsCustomerOrdersFrame"] = {
		DriftDisabledBy = "windowsDisabled",
	},
	["GuideFrame"] = {
		DriftDisabledBy = "windowsDisabled",
	},
	["ProfessionsBookFrame"] = {
		DriftDisabledBy = "windowsDisabled",
	},
}

```

**Comment by devwrk** [comment]
I kept getting an error for the minimap button. Might be a conflict with WarPlan and/or MasterPlan.
I ended up commenting those lines from Drift.lua without any obvious side effects other than fix my error....
```
--	["GarrisonLandingPageMinimapButton"] = {
--		DriftDisabledBy = "buttonsDisabled",
--	},
--	["ExpansionLandingPageMinimapButton"] = {
--		DriftDisabledBy = "buttonsDisabled",
--	},
```

**Comment by jaredbwasserman** [comment]
Taking a look.

**Comment by jaredbwasserman** [comment]
OK, thanks for the information, everyone. I made some fixes in Drift v2.3.0.

Drift v2.3.0 retail fixes:
1. The Interface Options issue is fixed.
2. The Professions Frame is movable.
3. The Talents Frame is movable.

For the minimap button errors @devwrk - I created this new issue https://github.com/jaredbwasserman/drift/issues/67 to track it.
